model/job-management/src/main/proto/beam_artifact_api.proto

```diff
@@ -81,7 +81,11 @@ message ProxyManifest {
 }
 
 // A request to get the manifest of a Job.
-message GetManifestRequest {}
+message GetManifestRequest {
+  // (Required) An opaque token representing the entirety of the staged artifacts.
```

Comment:
Maybe while we're at it let's rename it to retrieval_token? (here, in GetArtifactRequest, and in CommitManifestResponse; and also document in CommitManifestResponse that this token can be passed to GetManifestRequest and GetArtifactRequest)

Reply:
I missed this comment in my fixup, but I agree that now is a good time to rename this field.  Fixed and documented.

Reply:
FYI, this will need to be added to `JobInfo` in order to plumb this all the way through the workers. It also looks like you'll need to rebase #5584 with the rename.